### ISSUE TITLE
Set reponce option to False

### ISSUE THREAD

**Issue by mr-tz** [fireeye/flare-floss#142]
After `rep` issue is addressed (see https://github.com/williballenthin/viv-utils/pull/1) need to set the emulator option to False:
```
emu.setEmuOpt('i386:reponce', False)
```

**Comment by mr-tz** [comment]
See b15ce73 and 5f81c81

**Comment by mr-tz** [comment]
With repmax=0x100 the performance is back to a more reasonable level. For a test run on my machine I got:
```
32 passed, 22 xfailed in 572.11 seconds
```

**Comment by williballenthin** [comment]
`repmax` merged in viv-utils 0.3.2

**Comment by mr-tz** [comment]
Addressed in #128
